README.md

```diff
@@ -24,11 +29,15 @@ Those stories are generally created by the PM. If need be, developer can create
 Developers can assign stories to themselves and start working on them on a local branch.
 
 #In Progress
+When a story gets (self-)assigned, drag and drop the story into 'In Progress'
+
+#Needs Review
 As soon as the dev has code ready, a remote branch can be created and a new pull request created.
```

Comment:
As soon as the developer has code ready, a remote branch and a corresponding pull request can be created.